### DISCUSSION TITLE
Supporting CTRL + Z

### DISCUSSION THREAD

**Discussion by kas84** [revolist/revogrid#457] [General]
Hi, are you planning on supporting undo with CTRL+ Z?

**Comment by revolist** [comment]
Hello, this is Grid history feature, it's quite complicated but there is a future plan for it for sure.
Should be done as independent module.

**Comment by niklashaa** [comment]
Hi, are there any news on this?

**Comment by alexworldbr** [comment]
Hi,

This is a nice and important feature (ctrl Z and crtl R). 

I wish you can implement. 

Thanks.

**Comment by dennisgeerts** [comment]
Any further updates on this?

**Comment by m2a2x** [comment]
Hello @dennisgeerts 
at this moment we concentrated on new version for grid with advanced methods and plugin support.
This History feature would be release as a plugin later as Part of Pro features

**Reply by revolist** [reply]
Done as Pro
